hubblestack/extmods/modules/pulsar.py

```diff
@@ -395,3 +395,44 @@ def _dict_update(dest, upd, recursive_update=True, merge_lists=False):
             for k in upd:
                 dest[k] = upd[k]
         return dest
+
+#TODO:change paths for top.pulsar and config yaml to 'salt://hubblestack_pulsar/top.file'
+
+def top(topfile='/root/myhubblefiles/topfile/top.pulsar',
+        debug=None):
+
+    results_pulsar = []
+
+    configs = get_top_data(topfile)
+
+    if not isinstance(configs, list):
+        configs = configs.split(',')
```

Comment:
I don't think you need this `if` statement, because you know you'll always be returning a list from `get_top_data()`. I expect it was copy pasta.